hppc/src/main/java/com/carrotsearch/hppc/sorting/QuickSort.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * HPPC
+ *
+ * Copyright (C) 2010-2020 Carrot Search s.c.
```

Comment:
We should update the template (2020)....

Reply:
I tried 2021 but spotless corrected it :)